src/search/decorators/applyDecorators.ts

```diff
@@ -0,0 +1,21 @@
+import { SearchProduct, SearchResult } from "@nosto/nosto-js/client"
+import { HitDecorator, DecoratedProduct, DecoratedResult } from "../types"
+
+export function applyDecorators<HD extends HitDecorator[]>(response: SearchResult, decorators?: HD) {
```

Comment:
move file one level up?
since the actual decorators are in different modules
so this folder is a bit misleading